receiver/README.md

```diff
@@ -223,14 +231,27 @@ receivers:
 ## <a name="vmmetrics"></a>VM Metrics Receiver
 **Only metrics are supported.**
 
-<Add more information - I'm lonely.>
+Collects metrics from the host operating system. This is applicable when the
+OpenTelemetry Collector is running as an agent.
+```yaml
+receivers:
+  vmmetrics:
+    scrape_interval: 10s
+    metric_prefix: "testmetric"
+    mount_point: "/proc"
+    # process_mount_point: "/data/proc" # Should only be used for Daemon Set within a container.
```

Comment:
> Should only be used for Daemon Set within a container.

This isn't very clear.

Reply:
Updated slightly -- not an expert on this, appears to be related to differences between deployment models.